src/Icon/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './Icon.scss';
+
+function Icon(props) {
```

Comment:
you could also consider doing 
```javascript
function Icon({ id, className, hidden, srText })
```
and thus wouldn't have to write `props.X` everywhere.

Not a big deal either way!

Reply:
I find that the `props.X` is more clear especially with the format of other components but I get where you're coming from too.